radix-clis/src/replay/ledger_transaction.rs

```diff
@@ -58,11 +58,45 @@ impl RoundUpdateTransactionV1 {
 }
 
 impl TransactionPayload for RoundUpdateTransactionV1 {
-    type Versioned = FixedEnumVariant<{ TransactionDiscriminator::V1RoundUpdate as u8 }, Self>;
     type Prepared = PreparedRoundUpdateTransactionV1;
     type Raw = RawRoundUpdateTransactionV1;
 }
 
+// This shouldn't be implemented manually
```

Comment:
This part seems unclear.
`RoundUpdateTransactionV1` is not a variant of `VersionedTransactionPayload` and the comment does not really explain why we implement `SborEnumVariantFor` for `RoundUpdateTransactionV1`

Reply:
`RoundUpdateTransactionV1` is one of the node transactions which _should_ be a variant of `VersionedTransactionPayload` :) - it has a reserved discriminator value; it just lives in the node for historic / separation of concern reasons.

But I will move them to the engine in the next week or two and then we can remove most of this file :D

Reply:
Ok